src/App.tsx

```diff
@@ -54,7 +54,7 @@ export const App: React.FC = () => {
                 element={<Example3 />}
               />
               <Route
-                path="/add-certificate"
+                path="/add-certificate/:id?"
```

Comment:
please use enum AppRoutes.. this is not best approach but you can write `${AppRoutes.AddCertificate}/:id?`
and then the same on a place when navigating..